config/Shared/config_default.php

```diff
@@ -0,0 +1,173 @@
+    'generator' => [
+        'defaultConnection' => 'default',
+        'connections' => ['default', 'zed'],
+        'objectModel' => [
+            'defaultKeyType' => 'fieldName',
+            'builders' => [
+                // If you need full entity logging on Create/Update/Delete, then switch to
+                // Spryker\Zed\PropelOrm\Business\Builder\ObjectBuilderWithLogger instead.
+                'object' => ObjectBuilder::class,
+                'objectstub' => ExtensionObjectBuilder::class,
+                'query' => QueryBuilder::class,
+                'querystub' => ExtensionQueryBuilder::class,
+            ],
+        ],
+    ],
+    'paths' => [
+        'phpDir' => APPLICATION_ROOT_DIR,
+        'sqlDir' => APPLICATION_SOURCE_DIR . '/Orm/Propel/Sql/',
+        'migrationDir' => APPLICATION_SOURCE_DIR . '/Orm/Propel/Migration_SQLite/',
+        'schemaDir' => APPLICATION_SOURCE_DIR . '/Orm/Propel/Schema/',
+    ],
+];
+
+$config[PropelConstants::ZED_DB_ENGINE] = 'mysql';
+$config[PropelConstants::ZED_DB_HOST] = 'localhost';
+$config[PropelConstants::ZED_DB_PORT] = 1234;
+$config[PropelConstants::ZED_DB_USERNAME] = 'catface';
+$config[PropelConstants::ZED_DB_PASSWORD] = 'catface';
+
+$config[PropelConstants::PROPEL]['database']['connections']['default'] = $connections['mysql'];
+$config[PropelConstants::PROPEL]['database']['connections']['zed'] = $connections['mysql'];
+
+$config[KernelConstants::PROJECT_NAMESPACE] = 'Spryker';
+$config[KernelConstants::PROJECT_NAMESPACES] = ['Spryker'];
+$config[KernelConstants::CORE_NAMESPACES] = ['Spryker'];
+$config[KernelConstants::ENABLE_CONTAINER_OVERRIDING] = true;
+$config[ErrorHandlerConstants::ERROR_LEVEL] = E_ALL & ~E_DEPRECATED;
+$config[LogConstants::LOGGER_CONFIG] = SprykerLoggerConfig::class;
+$config[LogConstants::LOG_FILE_PATH] = sys_get_temp_dir() . '/logs';
+
+$config[AppPaymentConstants::APP_IDENTIFIER] = Uuid::uuid4()->toString();
+
+$config[MessageBrokerConstants::MESSAGE_TO_CHANNEL_MAP] =
+$config[MessageBrokerAwsConstants::MESSAGE_TO_CHANNEL_MAP] = [
+    PaymentAuthorizedTransfer::class => 'payment-events',
+    PaymentAuthorizationFailedTransfer::class => 'payment-events',
+    PaymentCapturedTransfer::class => 'payment-events',
+    PaymentCaptureFailedTransfer::class => 'payment-events',
+    PaymentRefundedTransfer::class => 'payment-events',
+    PaymentRefundFailedTransfer::class => 'payment-events',
+    PaymentCanceledTransfer::class => 'payment-events',
+    PaymentCancellationFailedTransfer::class => 'payment-events',
+    CancelPaymentTransfer::class => 'payment-commands',
+    CapturePaymentTransfer::class => 'payment-commands',
+    RefundPaymentTransfer::class => 'payment-commands',
+    AddPaymentMethodTransfer::class => 'payment-method-commands',
+    DeletePaymentMethodTransfer::class => 'payment-method-commands',
+    PaymentCreatedTransfer::class => 'payment-events',
+    // App event
+    AppConfigUpdatedTransfer::class => 'app-events',
+];
```

Comment:
has to be added to Readme file so app developer knows what has to be added to config_default.php of the app.